Get-NetView.psm1

```diff
@@ -227,6 +224,11 @@ function Open-GlobalThreadPool {
         [parameter(Mandatory=$true)] [Int] $BackgroundThreads
     )
 
+    if ($Scenario -eq "AzureHost") {
+        Write-Host "Using BackgroundThreads=0 for scenario $Scenario."
```

Comment:
Consider expanding this, such that the scenario display all modified behaviors encompassed.  See my comment on the scenario section for more details.

Reply:
For programmers, Ctrl+F the Scenario name to find all modified behavior.
For end-users, it's in the help-text.